src/style.css

```diff
@@ -0,0 +1,82 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  margin-top: 0;
+}
+
+@font-face {
+  font-family: "Avenir-Book";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir-Heavy";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 900;
+}
+
+.search-input {
+  position: relative;
+}
+
+.icon {
+  position: absolute;
+  width: 19px;
+  height: 19px;
+  left: 26px;
+  top: 0;
+}
+
+.input {
+  width: 100%;
+  padding-left: 62px;
+  color: #3d4e61;
+  font-family: Avenir-Book, sans-serif;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  background: #fff;
+  height: 70px;
+  font-size: 16px;
+}
+
+.icon--small {
+  width: 11px;
+  height: 11px;
+  top: 3px;
+  left: 13px;
+}
+
+.input:hover {
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+}
+
+.input:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+  font-family: Avenir-Heavy, sans-serif;
+  outline: none;
+}
+
+.input--small {
+  height: 42px;
+  font-size: 14px;
+  font-weight: 400px;
+  line-height: 19px;
+  padding-left: 33px;
+}
+
+.search-form {
+  height: 70px;
```

Comment:
why we need to define form height?

Reply:
![image](https://user-images.githubusercontent.com/97075591/204641264-523915ed-1e10-45c1-ac45-09ec8702096d.png)
without it i have next problem